circuit_maintenance_parser/parsers/tata.py

```diff
@@ -0,0 +1,69 @@
+# pylint: disable=disallowed-name
```

Comment:
could you apply it to a more specific line?

Reply:
It flagged on line 1 column 0. Would you mind advising where I could apply it? Thanks

> Run poetry run invoke pylint
LOCAL - Running command find . -name "*.py" | xargs pylint
************* Module circuit_maintenance_parser.parsers.tata
circuit_maintenance_parser/parsers/tata.py:1:0: C0104: Disallowed name "tata" (disallowed-name)